### DISCUSSION TITLE
#33 togETHer meeting - Feb 21 2022

### DISCUSSION THREAD

**Discussion by bobjiang** [tog-ETH-er/togETHer#8] [General]
Agenda:

- Community Updates
- Fireside discussion
- add `dapp learning` introduction to website

```
Dapp-Learing Project is suitable for developers with certain programming language foundation to start blockchain Dapp development. It allows developers to understand and develop various projects like Defi, NFT, DAO and CRYPTO from a lower level to the higher level step by step.  

The vision of the project is not only to provide junior developers an executable and simple learning roadmap for blockchain Dapp development, but to give advanced developers a platform to communicate and collaborate.


Dapp-Learing项目适合有一定语言基础的开发者入门区块链 DAPP 开发，由浅到深了解和开发 DeFi, NFT, DAO, CRYPTO 项目。

项目愿景是给初级开发者一个可执行且最简的区块链 Dapp 学习路线图，给进阶开发者一个可以交流和协作的平台。
```

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/s/LxNajKWJw4RV

会议 ID：891 5098 6775

手机一键拨号入会
+8675536550000,,89150986775#(中国大陆)
+85230018898,,,2,89150986775#(中国香港)

根据您的位置拨号
+8675536550000(中国大陆)
+85230018898(中国香港)

**Comment by luckyyang** [comment]
## Rebase
- Rostra: [sprint 5](https://rostra.notion.site/Sprint-5-c81f0df61a624179b89591f432bdbf89)
- web3 Daily: https://rebase.network/posts/category/web3daily

**Comment by 0x-stan** [comment]
# Dapp-Learning

近两周工作进展

## repo update

21个PR #584 ～ #608

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了3场分享会，其中《On the Future of Web3》是我们的首场全英文分享会，而《DAO spectrum》是由国际友人带来的中文分享。

- On the Future of Web3 
  - by Qizhou
  - https://www.youtube.com/watch?v=O4j1W8uh_hE&
  - https://www.youtube.com/watch?v=5gbPAwyP8Z0
- TaiShangWorldGenerator项目介绍
  - by Leeduckgo
  - https://www.youtube.com/watch?v=ylRmNvz--6Y
- DAO Spectrum
  - by Justinzhang
  - https://www.youtube.com/watch?v=6KsdZyB8nZ8

 
## 翻译小组

- MIT公开课翻译第9集发布
  - https://mp.weixin.qq.com/s/4DfIhqKjKSZb49lD2UlqhQ
- MIT公开课翻译第10集发布
  - https://mp.weixin.qq.com/s/vzddMUW5ffsN-STWrpgWlA
- MIT公开课翻译第11集进入审稿阶段

**Comment by Emilyxiannv** [comment]
## DAOSquare
DAOSquare 做了一期关于Ceramic的分享，视频中用通俗易懂的语言向大家展示了Ceramic 作为Web3 时代的去中心化存储的概念，优势，实现逻辑以及DAOSquare的投资理由等；
[Ceramic.pdf](https://github.com/tog-ETH-er/togETHer/files/8108868/Ceramic.pdf)

**Reply by Emilyxiannv** [reply]
https://self.id/

有兴趣可以去体验一下，这是我们个人用户的一个小小的使用场景

**Reply by Emilyxiannv** [reply]
视频链接：https://www.youtube.com/watch?v=dGirQEOs9eo&t=262s ；https://www.youtube.com/watch?v=QZd51ZAakbY

**Comment by stephanieeee-ho** [comment]
# ECN
1. 常规翻译和七日谈
2. 春节活动的收尾

**Comment by Riely007** [comment]
### 以太坊翻译小组

1. 继续 ethereum.org 的翻译，[目前进度 98%](https://crowdin.com/project/ethereum-org?already_accepted=14553216) ；
2. 梳理 eip 翻译的进度

**Comment by bobjiang** [comment]
Bob

1. GitcoinDAO dev $100/h
2. ENS ecosystem wg
3. Beijing meetup on Feb 27

**Comment by Jackieyewang** [comment]
1. 组建了web3实习生群
2. 组织了DeFi的讨论twitter space

**Comment by winkybupt** [comment]
# 原语里弄
一个新课题完成开题；
邀请 Crypto Tech Night 加入。

**Comment by zengmi2140** [comment]
## Ethfans:
历史文章整理：https://github.com/editor-Ajian/EthFans.org-annual-collected-works
